src/impf.cpp

```diff
@@ -1031,10 +1033,6 @@ x		}
 	}
 	return rc;			// set to RCBAD iff err() calls above
 
-// bad header record read common error exit
```

Comment:
What was the reason for removing this particular label and goto code? I'm not sure that multiple returns in the middle of a function are beneficial.